pkg/scheduler/tests/scheduler_smoke_test.go

```diff
@@ -936,3 +938,136 @@ partitions:
 
     waitForAcceptedApplications(mockRM, "app-added-2", 1000)
 }
+
+func TestSchedulingOverCapacity(t *testing.T) {
+    serviceContext := entrypoint.StartAllServicesWithManualScheduler()
+    defer serviceContext.StopAll()
+    proxy := serviceContext.RMProxy
+
+    // Register RM
+    configData := `
+partitions:
+  -
+    name: default
+    queues:
+      -
+        name: root
+        submitacl: "*"
+        queues:
+          -
+            name: default
+            properties:
+              application.sort.policy: fair
+            resources:
+              guaranteed:
+                memory: 100
+                vcore: 10
```

Comment:
Can we minimise the config as much as possible?
We do not use the properties or guaranteed resources in the test.

Reply:
Yep, we can do that. Fixed in the commit-2.